security/aporeto/build/ansible/group_vars/prod.yml

```diff
@@ -1,9 +1,57 @@
-aporeto_release: 3.11.9
+# Aporeto Specific Details
+aporeto_release: 3.11.15
+aporeto_enforcer_memory_limit: '4Gi'
```

Comment:
to be on the safe side, I'd like to increase this to 8Gi for prod. We can tune this down if we need to.